default/python/stub_generator/parse_docs.py

```diff
@@ -164,9 +186,9 @@ def parse_signature(line):
         self.resources = res
 
         args, rtypes, infos = zip(*res)
-        rtypes = set(rtypes)
-        assert len(rtypes) == 1, "Different rtypes for: %s in: %s" % (rtypes, text)
-        self.rtype = normilize_rtype(rtypes.pop())
+        if len(set(rtypes)) != 1:
+            error("[%s] Different rtypes", name)
+        self.rtype = normilize_rtype(rtypes[0])
```

Comment:
Since we are here, please also correct the spelling of this function name to ```normalize_rtype```

I also see a number of places where 'routine' is misspelled as 'rutine'; it's all done consistently so it doesn't cause errors, but it would be nice to get that corrected also.